nextflow.config

```diff
@@ -18,6 +18,8 @@ params {
     checkm_metadata             = null
     gtdbtk_metadata             = null
     se_reads                    = false
+    check_duplicates            = true
+    rename_contigs              = false
```

Comment:
I wonder about this one, wouldn't a default of true actually be better? We can issue a warning from the script if there are duplicates that will get renamed and, in that warning, note that there's an option to turn off this.